File: app/src/main/java/com/softartdev/notecrypt/ui/note/NoteView.java
package com.softartdev.notecrypt.ui.note;

interface NoteView {
    void onLoadNote(String title, String text);
    void onSaveNote(String title);
    void onEmptyNote();
    void onDeleteNote();
    void onCheckSaveChange();
    void onNavBack();
}

File: app/src/main/java/com/softartdev/notecrypt/ui/note/NoteView.kt
package com.softartdev.notecrypt.ui.note

import com.softartdev.notecrypt.ui.base.MvpView

interface NoteView : MvpView {
    fun onLoadNote(title: String, text: String)
    fun onSaveNote(title: String)
    fun onEmptyNote()
    fun onDeleteNote()
    fun onCheckSaveChange()
    fun onNavBack()
}
